src/collections/members/bariq-nurlis/index.mdx

```diff
@@ -0,0 +1,13 @@
+---
+name: Bariq Nurlis
+position: Contributor
+image_path: ./bariq-nurlis.jpeg
+github: bariqhibat
+linkedin: bariqhibat
+location: Singapore/Indonesia
+bio: Bariq is an undergraduate student who currently studying engineering in NTU Singapore. He is a passionate maker who likes to build things and watch them benefitting people's lives. He's excited to contribute and learn in the open source community.
+badges:
```

Comment:
Meshery?

Reply:
cc @leecalcote

Reply:
Yes, "meshery" make sense. 👍

Reply:
Yes, I think it'd be good to put Layer5/Meshery instead 

// @sudo-NithishKarthik